frontend/app/routes.json

```diff
@@ -939,6 +939,14 @@
                   }
                 ]
               },
+              {
+                "id": "public/renew/$id/adult-child/review-information",
+                "file": "routes/public/renew/$id/adult-child/review-information.tsx",
```

Comment:
Nitpick: I think the id/file should be `review-adult-information` to be consistent with the `apply` flow naming conventions.